src/config/external-applications.md

```diff
@@ -0,0 +1,51 @@
+# External Applications
+
+## Programming Languages
+
+> The Void repositories have a number of python and lua packages. If possible,
+> try to install packages from the Void repos using xbps-install or use xbps-src
+> to package your python or lua application. Packaging your application with
+> xbps-src will keep your system clean. If you send your package upstream it
+> will allow other Void users to benifit and allow them to help you maintain
+> your package. To learn how to package for xbps-src see the
+> [manual](https://github.com/void-linux/void-packages/blob/master/Manual.md)
+> and [contributing
+> instructions](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md).
+
+To keep your system lean, Void splits packages and their devel packages. If you
+install a program via the languages package manager (pip, gem, etc.) or compile
+via source you may need to install the programming language's devel package to
+ensure your program builds.
+
+| Language | Package Manager                | Void Package  |
+|----------|--------------------------------|---------------|
+| Python   | pip, anaconda, virtualenv, etc | python3-devel |
+| Ruby     | gem                            | ruby-devel    |
+| lua      | luarocks                       | lua-devel     |
+
+## Restricted Packages
+
+Some packages have legal restrictions on their distribution (ie. discord), may
+be too large, or have another condition that makes it difficult for the void
+team to distribute. These packages are maintained but not distributed. As such,
+they must be built locally. For more information see the page on [restricted
+packages](../repositories/restricted.md).
+
+## Non-x86_64 Arch
+
+The entire Void build system is using the x86_64 architecture. xbps-src allows
+us to cross-compile for other architectures and their glibc or musl variant.
+However, some packages (ie. libreoffice) do not support cross-compilation. For
+these appplications, you must build the application locally on the machine you
+want to use the package. For compiling and installing xbps-src applications see
+the [building instructions](../packages/building.md).
+
+## Flatpak
+
+Flatpak is another method for installing external proprietary applications on
+linux. For information on using flatpak with Void Linux, see the [official
+flatpak documentation](https://flatpak.org/setup/Void%20Linux/).
+
+> Flatpak's sandboxing will not protect you from the malicious features
+> proprietary applications may have. Whenever possible, try to use free software
+> with Void Linux.
```

Comment:
I really like this paragraph, I just don't know if it fits the tone we want to have on the docs.

@bobertlo what do you think?

Reply:
i'm not sure the last sentence is necessary; i think it would be sufficient to just say something like "Flatpak's sandboxing will not necessarily protect you from any security- and/or privacy-violating features of proprietary software".

Reply:
I think proprietary applications usually have more malicious features than just merely security issues and privacy violations.

ie: vendor lock-in, censorship, deception, using one software to enforce the use of another one of your corporation's offerings.

It's more accurate to mention malicious features or anti-features.

Reply:
i don't disagree with your points in general - although i would disagree with any implication that FOSS never has anti-features - but the Handbook is technical documentation, and i'm concerned about starting down a steep slippery slope. i'd prefer that we didn't set a precedent for the Handbook to become a place where people argue over licensing issues. So i think that what we need to focus on is letting people know that 'sandboxing' is not a magical protective incantation.

(i regularly encounter people who think that merely using OpenBSD means their system is secure, regardless of what software they install and use.)

Reply:
That is a good point. If anyone has a recommendation to change the wording such that the tone is better while ensuring users know that sandboxing is not magic, I will accept it.

Reply:
@anjandev, I really like flexibeast's wording.

> Flatpak's sandboxing will not necessarily protect you from any security- and/or privacy-violating features of proprietary software.

It's short and sweet.

Reply:
Changed to @flexibeast's wording